src/main/java/com/iridium/iridiumteams/database/IridiumUserProfile.java

```diff
@@ -0,0 +1,89 @@
+package com.iridium.iridiumteams.database;
+
+import com.iridium.iridiumteams.IridiumTeams;
+import com.iridium.iridiumteams.enhancements.*;
+import com.j256.ormlite.field.DatabaseField;
+import com.j256.ormlite.table.DatabaseTable;
+import lombok.Getter;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+import org.bukkit.scheduler.BukkitTask;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.LocalDateTime;
+import java.util.*;
+
+@Getter
+@DatabaseTable(tableName = "users_profile")
+public class IridiumUserProfile<T extends Team> extends DatabaseObject {
+
+    @DatabaseField(columnName = "id", canBeNull = false, generatedId = true)
+    private int id;
+
+    @DatabaseField(columnName = "user_uuid", canBeNull = false)
+    private @NotNull UUID uuid;
+
+    @DatabaseField(columnName = "profile_name", canBeNull = false)
+    private @NotNull String name;
+
+    @DatabaseField(columnName = "team_id")
+    private int teamID;
+    @DatabaseField(columnName = "user_rank", canBeNull = false)
+    private int userRank;
+
+    @DatabaseField(columnName = "join_time")
+    private LocalDateTime joinTime;
```

Comment:
createTime*

Reply:
nevermind, thats the time they joined the island. might also be worth having a create time field, for the time the profile was created?